types/focus-within/test/focus-within-import-default-tests.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * This file tests the typings of the default export of `focus-within`
+ */
+import focusWithin from 'focus-within';
```

Comment:
this should be `import focusWithin = require("focus-within");` to ensure it works in older versions of TS that the tester uses.